### PR TITLE
chore: update edge integrations to use FGAC tokens

### PR DESCRIPTION
**Pull request by pratik151192**
Updating all edge integration docs to use FGAC tokens and not super-user token. The caches are already created in each of the examples and the code is simply doing a get/set.

### CODE REVIEW

**Comment by pratik151192** [comment]
Added issue to do a generic sweep https://github.com/momentohq/public-dev-docs/issues/367

File: docs/develop/integrations/cloudflare.md
Comment (NoSQLKnowHow) [inline docs/develop/integrations/cloudflare.md]:
```suggestion
Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure you choose the same region when generating a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage the security of control plane and data plane operations separately. 
```

File: docs/develop/integrations/vercel-integration.md
Comment (NoSQLKnowHow) [inline docs/develop/integrations/vercel-integration.md]:
```suggestion
Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure you choose the same region when generating a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage security of control plane and data plane operations separately.
```

File: docs/develop/integrations/netlify-integration.md
Comment (NoSQLKnowHow) [inline docs/develop/integrations/netlify-integration.md]:
```suggestion
Notice the region you created your cache in is also displayed in the list of caches. You'll need to make sure you choose the same region when generating a Momento authentication token in the next step. Navigate to the [tokens](https://console.gomomento.com/tokens) page, and choose the cloud provider and region you used to create your cache. Since the cache is already created, we will use a fine-grained token that will allow the worker to read from and write to the cache; but will not allow it to do control plane operations such as delete. This is especially helpful if you want to manage the security of control plane and data plane operations separately.
```

**Review by NoSQLKnowHow — DISMISSED** [review]

**Comment by pratik151192** [comment]
@NoSQLKnowHow thanks! waiting for @pgautier404 approval; he just wants to double check the netlify/vercel integrations won't break with the FGAC tokens.

File: docs/develop/integrations/netlify-integration.md
Comment (pgautier404) [inline docs/develop/integrations/netlify-integration.md]: This app isn't reading or writing to cache. It's 100% pubsub and the cache has already been created, so no need to discuss control or data plane in this context really. These instructions need to be updated to describe how to generate a `publishsubscribe` topics token for all topics in the `chat` cache.

File: docs/develop/integrations/vercel-integration.md
Comment (pgautier404) [inline docs/develop/integrations/vercel-integration.md]: Same as above.

File: docs/develop/integrations/netlify-integration.md
Comment (pgautier404) [reply docs/develop/integrations/netlify-integration.md]: Scratch that :-) These chat apps require a superuser token because they generate expiring FGAC tokens to the browsers. The docs are correct as written originally.

File: docs/develop/integrations/vercel-integration.md
Comment (pgautier404) [reply docs/develop/integrations/vercel-integration.md]: See above again :-)

**Review by pgautier404 — CHANGES_REQUESTED** [review]
Need to revert changes to Vercel and Netlify as they require a superuser token to function.

File: docs/develop/integrations/vercel-integration.md
Comment (pratik151192) [reply docs/develop/integrations/vercel-integration.md]: Done

File: docs/develop/integrations/netlify-integration.md
Comment (pratik151192) [reply docs/develop/integrations/netlify-integration.md]: Done

**Review by pgautier404 — APPROVED** [review]

**Pull request merged**
